config/default.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
   },
 
   cors: {
-    allowedOrigins: /(https:\/\/([a-zA-Z0-9-_]+\.)?realdevsquad\.com$)/, // Allow realdevsquad.com, *.realdevsquad.com
+    allowedOrigins: /(http:\/\/localhost:[0-9]{4}$)/, // Allow realdevsquad.com, *.realdevsquad.com
```

Comment:
Seems wrong, especially according to the comment. We don't want our CORS policy to be open to localhost by default.

Reply:
@khalatevarun I think the custom configs can be put inside a `config/local.js` file where it would still work as it is in `default.js` while also being ignored by git when committing your changes.

Reply:
reverted this change